DESCRIPTION

```diff
@@ -44,6 +45,7 @@ Imports:
     lifecycle
 Suggests:
```

Comment:
```suggestion
Remotes:
    larmarange/broom.helpers
Suggests:
```

Reply:
I'm ok having a remote on the master branch, if you'd like to add this to the DESCRIPTION

Reply:
Merging now to help checks

Reply:
Dear @schloerke  

I'm not sure about using a remote. Soon a version 1.1.0 of `broom.helpers` will be released on CRAN. I'm thinking that it would be better to have ggcoef_model merged into GGally at that time.

Reply:
Or is it just for the time of testing the PR?

Reply:
Just for the testing of the PR. Makes everything a little smoother. Will remove the `Remotes` once it's on CRAN or before the next release of GGally.